styles.css

```diff
@@ -0,0 +1,65 @@
+* {
+    box-sizing: border-box;
+  }
+  
+  html {
+    
+      font-size: 10px;
+    }
```

Comment:
Good job on the css use! I just recommend you to change the identation, remember that you should use 2 spaces to ident your code to avoid any linters issue and use the best css practices.

Reply:
- Good use of inline comment and polite phrases, Keep it up